### DISCUSSION TITLE
Best/proper way for pairing new devices?

### DISCUSSION THREAD

**Discussion by m0wlheld** [Koenkk/zigbee2mqtt#5710] [General]
Dear all,

I'm about to introduce 25 new light bulbs ([Innr RB265](https://www.zigbee2mqtt.io/devices/RB_265.html) and [Trådfri LED1836G9](https://www.zigbee2mqtt.io/devices/LED1836G9.html)) to my Zigbee network. 

What is the proper way of joining:
* pair each bulb with the coordinator (RPi4B, Raspbee II, zigbee2mqtt and Home Assistant), then move it to it's final location OR
* pair bulbs one by one with the closest routing device

The first bulb is within range of the coordinator (~ 10m), the last one is about 150m away.

**Comment by Kryzek** [comment]
Personally I pair devices near or at the location it will be used. Every device I have has been working nicely with this approach, I have Philips, Ikea, Lidl, Xiaomi and Airam (and some random ones).

**Comment by m0wlheld** [comment]
So did I, worked like a charm, but only Innr bulbs used today. 20 Trådfri still waiting for first use.

**Reply by m0wlheld** [reply]
With Tradfri, I had to do a factory reset first and paired them close to the coordinator, as written on the [device support page](https://www.zigbee2mqtt.io/devices/LED1836G9.html). When installed at the final location, it took less then a minute for them to reconnect.
